### PR TITLE
Release notes update regarding logs

### PR DESCRIPTION
**Pull request by marians**
Ping @luebken @ewoutp - Please review

### CODE REVIEW

File: content/reference/release-notes.md [hunk 1]
```diff
@@ -15,8 +15,9 @@ Here we list the more important changes we made between releases. See our [insta
 Released 2015-05-11
 
 * The new function `swarm exec` allows to execute arbitrary commands in a running component instance, including opening a shell. Read more about it on the according [reference page](/reference/exec/).
+* Logs are now retained for a much longer period (currently 14 days), as long as the component instances still exists.
```
Comment (marians) [inline content/reference/release-notes.md]: OK

File: content/reference/release-notes.md [hunk 2]
```diff
@@ -15,8 +15,9 @@ Here we list the more important changes we made between releases. See our [insta
 Released 2015-05-11
 
 * The new function `swarm exec` allows to execute arbitrary commands in a running component instance, including opening a shell. Read more about it on the according [reference page](/reference/exec/).
+* Logs are now reliably retained for a much longer period (currently 14 days), as long as the component instances still exist.
```
Comment (ewoutp) [inline content/reference/release-notes.md]: This sentence is technically completely correct, but it gave me the "expectation" that logs were retained for 14 days, not just for as long as the instance is alive.

I think I had the tendency to stop reading before the second halve of the sentence.

Comment (ewoutp) [inline content/reference/release-notes.md]: Don't know if or how this should/could be altered

**Comment by ewoutp** [comment]
Technically correct, so LGTM

**Comment by luebken** [comment]
LGTM

File: content/reference/release-notes.md (hunk 2 above)
Comment (marians) [inline content/reference/release-notes.md]: Please check again. I also added a change to the `swarm logs` reference page in this PR.

**Comment by marians** [comment]
Tried to make it more clear. Also added a change to the swarm logs reference page, which I had previously forgotten.

File: content/reference/release-notes.md [hunk 3]
```diff
@@ -15,8 +15,9 @@ Here we list the more important changes we made between releases. See our [insta
 Released 2015-05-11
 
 * The new function `swarm exec` allows to execute arbitrary commands in a running component instance, including opening a shell. Read more about it on the according [reference page](/reference/exec/).
+* For any component that creates log output, log data is now reliably retained for a fixed period (currently 14 days) as long as the source component instance still exists. Deleting a component instance makes the according log data inaccessible for you as a user.
```
Comment (ewoutp) [inline content/reference/release-notes.md]: I would remove the "for you as a user" part, otherwise I like it.

Comment (marians) [inline content/reference/release-notes.md]: OK, done

**Comment by marians** [comment]
Thanks for reviewing! Will merge now.

**Pull request merged**